todo.sh

```diff
@@ -1485,14 +1485,22 @@ note: PRIORITY must be anywhere from A to Z."
 "listaddons" )
     if [ -d "$TODO_ACTIONS_DIR" ]; then
         cd "$TODO_ACTIONS_DIR" || exit $?
+        VALID=0
```

Comment:
Use a lowercase variable (yeah, it's not consistent yet), and maybe `actionsCnt`, as it's not a boolean flag.